app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MailerHelper
+  def email_footer
+    uuid = Setting[:instance_id]
+    title = Setting[:instance_title]
+
+    if uuid.present? && title.present?
+      _('This email was sent from Foreman instance %{title} and with identifier %{uuid}') % {title: title, uuid: uuid}
+    elsif uuid.present?
+      _('This email was sent from Foreman instance with UUID %s') % uuid
```

Comment:
can we keep the same format for the sprint?
```suggestion
      _('This email was sent from Foreman instance with UUID %{uuid}') % { uuid: uuid }
```